plugins/cq/commands/status.md

```
@@ -24,24 +24,28 @@ local: {count} | private: {count} | public: {count}
 ■ 0.3-0.5: {count} units
 ■ 0.0-0.3: {count} units
+
+### Domains
+{distinct count} total — {domain} ({count}), {domain} ({count}), ... +{remaining} more
 ```
 
 The `tier_counts` field contains the tier breakdown. Display all tiers present in the response. Omit tiers with a count of 0.
 
 The `recent` field reflects the local store only. When a remote is configured and reachable, units proposed via `Client.Propose` go directly to the remote and do not appear here. If `recent` is empty, render the section as `(no recent local additions)` so users understand the scope.
 
+The `domain_counts` field maps each domain to its unit count. Domains are the least important section, so render them last and keep them compact: lead with the number of distinct domains, then the most-tagged few inline as `{domain} ({count})` (highest count first, ties alphabetical), capped at 8. If more than 8 exist, append ` ... +{remaining} more`; do not leave a trailing comma before the ellipsis.
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)